src/utils/constants.ts

```diff
@@ -255,4 +269,29 @@ export default {
   TEMPLATE_BUTTON_SAVE: 'Save',
   TEMPLATE_BUTTON_ADD: 'Add',
   TEMPLATE_APPEARS_IN: 'Appears in',
+  OUTCOMES_TABLE_PAGINATION_TEXT: 'Items per page',
+  APP_LINK_LINKED_APPLICATIONS: 'Linked Applications',
+  APP_LINK_NAME: 'Name',
+  APP_LINK_SERIAL: 'Serial',
+  APP_LINK_TYPE: 'Application Type',
+  APP_LINK_COMPLETED: 'Date Completed',
+  LOOKUP_ERROR_NOT_FOUND: 'No lookup-table found!',
+  LOOKUP_ERROR_TITLE: 'Error loading lookup-table',
+  LOOKUP_TABLE_HEADER_LABEL: 'Label',
+  LOOKUP_TABLE_HEADER_NAME: 'Table Name',
+  LOOKUP_TABLE_HEADER_ACTIONS: 'Actions',
+  LOOKUP_TABLE_VIEW: 'View lookup table',
+  LOOKUP_TABLE_IMPORT: 'Import lookup table',
+  LOOKUP_TABLE_IMPORT_INTO: 'Import into Lookup-table: %1',
```

Comment:
Substitutions (%1) are fine within sentences, but I think we should keep it to a minimum and don't put a substitution in a localised string if possible. In this case, it's unnecessary -- the string should just be "Import into look-up table" and the ": {table name}" part should be purely in the code and not something the translator should have to think about.

Reply:
Ok - cool! I'll be changing this as well : )